src/redux/actions/missionsActions.js

```diff
@@ -0,0 +1,18 @@
+export const SET_MISSIONS = 'SET_MISSIONS';
+export const TOGGLE_MISSION = 'TOGGLE_MISSION';
+
+export const setFetchedMissions = (missions) => ({
+  type: SET_MISSIONS,
+  missions,
+});
+
+export const fetchMissions = () => (dispatch) => {
+  fetch('https://api.spacexdata.com/v3/missions')
+    .then((response) => response.json())
+    .then((data) => dispatch(setFetchedMissions(data)));
```

Comment:
- [ ] Currently, upon fetching the data, your state stores unnecessary data. As stated in the requirements only the selected API data (`mission_id`, `mission_name`, `description`) should be dispatched to the store. Hence you should ensure the data from the API is filtered and only the correct ones are saved to the store. This is correctly done in the  `src/redux/actions/ReduxActions.js` file.

  Data to dispatch | Your dispatched data
  :---:|:---:
  ![reem-lab-missions-data](https://user-images.githubusercontent.com/36542986/156485175-cd41b707-87bb-472b-8eb5-6a4123a66caa.jpg) | ![reem-lab-missions](https://user-images.githubusercontent.com/36542986/156484835-88e85bad-c452-47b0-863d-66ddc3a76b2e.jpg)